specification/synapse/resource-manager/Microsoft.Synapse/preview/2019-06-01-preview/sqlPool.json

```diff
@@ -3833,6 +3833,174 @@
           }
         }
       }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Synapse/workspaces/{workspaceName}/sqlPools/{sqlPoolName}/transparentDataEncryption/{transparentDataEncryptionName}/operationResults": {
```

Comment:
You definitely don't need to (and shouldn't) list async operationResults in your SDK or API.  You typically don't need to expose operationResults at all in the SDK since the SDKs will automatically follow operationResults links from async operations.

Reply:
Ah, I see this has nothing to do with async operations.  It should be renamed so it isn't using the same resource type name as async operation results (https://github.com/Azure/azure-resource-manager-rpc/blob/master/v1.0/Addendum.md#202-accepted-and-location-headers)

Reply:
We've based this change off of swagger documentation for this:

https://docs.microsoft.com/en-us/rest/api/sql/transparentdataencryptionactivities/listbyconfiguration

"/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Synapse/workspaces/{workspaceName}/sqlPools/{sqlPoolName}/transparentDataEncryption/{transparentDataEncryptionName}/activities? 

Does the above look ok ?

Reply:
Yes, thank you

Reply:
As a result of some confusion in the Synapse RP side, with this Transparent dataencryption activities /operationResults API, we've decided to hold off on this change. So resolving this comment as this code is not going to be there anymore in the PR